src/ssl.cpp

```diff
@@ -792,48 +792,68 @@ int SSL_CTX_load_verify_locations(SSL_CTX* ctx, const char* file,
         WIN32_FIND_DATA FindFileData;
         HANDLE hFind;
 
-        char name[MAX_PATH + 1];  // directory specification
-        strncpy(name, path, MAX_PATH - 3);
+        char* name = NEW_YS char[strlen(path) + 4];  // directory specification
+        int nameSz = strlen(path) + 4; // 4 for extra space needed with string
+        strncpy(name, path, nameSz - 3);
```

Comment:
Magic numbers should be removed.   These should have names.  DIR_SEP_SZ, DIR_SEP_STAR_SZ?  +1 can denote terminator but the comment should reflect that.